docs/articles/expensify-classic/workspace-and-domain-settings/Categories.md

```diff
@@ -1,5 +1,137 @@
 ---
 title: Categories
-description: Categories
+description: Categories are used to classify and organize expenses
 ---
-## Resource Coming Soon!
+# Overview
+Categories are commonly used to classify and organize expenses for both personal finances and business accounting.
+The way Categories function will vary depending on whether or not you connect Expensify to a direct account integration (i.e., QuickBooks Online, NetSuite, etc.). 
+When reviewing this resource, be sure to take a look at the section that applies to your account setup! 
+# How to use Categories 
+- When using an accounting integration, categories are your chart of accounts/ show in expense claims/etc. 
+- You can have different categories for different workspaces
+# How to import Categories (no accounting integration connected)
+## Add Categories via spreadsheet
+If you need to import multiple categories at once, you can upload a spreadsheet of these parameters directly to Expensify. 
+Before importing the category spreadsheet to Expensify, you'll want to format it so that it lists the Category name as well as any optional fields you'd like to include.
+Required fields: 
+- Category name
+Optional fields: 
+- GL Code
+- Payroll code
+- Enabled (TRUE/ FALSE)
+- Max Expense amount
+- Receipt Required
+- Comments (Required/ Not Required)
+- Comment Hint
+- Expense Limit Type
+  
+Expensify supports the following file formats for uploading Categories in bulk:
+
+- CSV
+- TXT 
+- XLS 
+- XLSX
+  
+Once the spreadsheet is formatted, you can upload it to the workspace under Settings > Workspace > Group >  Workspace Name > Categories. 
```

Comment:
Missing formatting from the doc:

**Settings > Workspace > Group >**  _Workspace Name_ **> Categories**

Add new line after sentence.

Reply:
Formatting is not correct, now it's all in bold whereas it should be with _Workspace Name_ in italic.

Reply:
I resolved this.